theano/sandbox/cuda/dnn.py

```diff
@@ -1972,28 +1966,28 @@ def infer_shape(self, node, shape):
         return [shape[0]]
 
 
-def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
+def dnn_pool(img, ws, stride=None, mode='max', pad=None):
     """
     GPU pooling using cuDNN from NVIDIA.
 
-    The memory layout to use is 'bc01', that is 'batch', 'channel',
-    'first dim', 'second dim' in that order.
+    For 2D pooling, the memory layout to use is 'bc01', that is 'batch',
+    'channel', 'first dim', 'second dim' in that order.
+
+    For 3D pooling, the memory layout to use is 'bc012', that is 'batch',
+    'channel', 'first dim', 'second dim', 'third dim'.
 
     Parameters
     ----------
     img
         Images to do the pooling over.
     ws
-        Subsampling window size.
+        Subsampling window size.  Should have 2 or 3 elements.
     stride
-        Subsampling stride (default: (1, 1)).
+        Subsampling stride (default: (1, 1) or (1, 1, 1)).
     mode : {'max', 'average_inc_pad', 'average_exc_pad}
-    pad :
-        (pad_h, pad_w) padding information.
-        pad_h is the number of zero-valued pixels added to each of the top and
-        bottom borders.
-        pad_w is the number of zero-valued pixels added to each of the left
-        and right borders.
+    pad
+        Padding information.  A tuple with 2 or 3 integers, specifying the
+        number of zero-valued pixels to be added on both sides of the input.
```

Comment:
Can you keep the extra information from the old description? It was telling which element of the tuple correspond to which dimensions.